tasks/testwasher.py

```diff
@@ -229,6 +224,7 @@ def generate_flake_finder_pipeline(ctx, n=3):
                 ):
                     new_job['variables']['E2E_COMMIT_SHA'] = "$PARENT_COMMIT_SHA"
             new_job["rules"] = [{"when": "always"}]
+            new_job["needs"] = ["!reference [.needs_new_e2e_template]"]
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Does this work ? This is an array containing the string `.needs_new_e2e_template` tagged with `reference` != just a string

Reply:
Isn't `needs` an array? https://docs.gitlab.com/ee/ci/yaml/#needs

Reply:
discussed offline! We changed the way to do it